TYPO3.Neos/Classes/TYPO3/Neos/Utility/BackendAssetsUtility.php

```diff
@@ -39,7 +39,7 @@ public function injectSettings(array $settings)
      */
     public function shouldLoadMinifiedJavascript()
     {
-        return isset($this->settings['userInterface']['loadMinifiedJavaScript']) ? $this->settings['userInterface']['loadMinifiedJavaScript'] : $this->settings['userInterface']['loadMinifiedJavascript'];
+        return isset($this->settings['userInterface']['loadMinifiedJavaScript']) ? $this->settings['userInterface']['loadMinifiedJavaScript'] : true;
```

Comment:
I guess we're talking cross purpose here.. What I was trying to say: We ignore invalid configuration mostly (that's what we have the configuration schema validator for).
But once we remove support for a setting that was previously supported (like this PR would do) we usually show a helpful error message